extensions/omni_httpd/event_loop.c

```diff
@@ -324,19 +337,35 @@ static void on_message(h2o_multithread_receiver_t *receiver, h2o_linklist_t *mes
       break;
     }
     case MSG_KIND_UPGRADE_TO_WEBSOCKET: {
-      h2o_websocket_conn_t *websocket_conn =
-          h2o_upgrade_to_websocket(req, reqmsg->ws_client_key, reqmsg, on_ws_message);
-      websocket_conn->data = malloc(sizeof(reqmsg->ws_uuid));
-      memcpy(websocket_conn->data, reqmsg->ws_uuid, sizeof(reqmsg->ws_uuid));
+      on_ws_message_data_t *data = malloc(sizeof(on_ws_message_data_t));
+      // if (!data) ???
```

Comment:
Allocation failure? Well, we can't proceed  from there so I guess we just log a WARNING and abort?

Reply:
I wasn't sure how to log it. Just print to stderr?

Everything else logs to postgres by this call is running in the h2o thread and does not have access to anything postgres directly and probably shouldn't as I haven't seen anything to suggest that it is threadsafe?